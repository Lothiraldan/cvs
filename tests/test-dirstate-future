#!/bin/sh

hg init
echo a > a
hg add
hg ci -m1

# set mtime of a into the future
touch -t 202101011200 a

# status must not set a's entry to unset (issue1790)
hg status
hg debugstate
