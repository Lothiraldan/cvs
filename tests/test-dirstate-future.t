Prepare test repo:

  $ hg init
  $ echo a > a
  $ hg add
  adding a
  $ hg ci -m1

Set mtime of a into the future:

  $ touch -t 202101011200 a

Status must not set a's entry to unset (issue1790):

  $ hg status
  $ hg debugstate
  n 644          2 2021-01-01 12:00:00 a

