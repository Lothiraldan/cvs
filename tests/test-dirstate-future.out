adding a
n 644          2 2021-01-01 12:00:00 a
